app/models/conference.rb

```diff
@@ -95,9 +95,11 @@ def confirmed
 
   validates_uniqueness_of :short_title
   validates_format_of :short_title, with: /\A[a-zA-Z0-9_-]*\z/
+  validates :registration_limit, numericality: { only_integer: true, greater_than_or_equal_to: 0 }
 
   # This validation is needed since a conference with a start date greater than the end date is not possible
   validate :valid_date_range?
+  validate :registration_limit_not_exceed
```

Comment:
Aha, and now you see that the validation belongs to the registration and not to the conference! Because when the limit is reached you shouldn't be able to create a registration / the registration should be invalid. With your test you're able to create the registration though it isn't valid, instead the conference is now invalid. 

So, can you please move the validation to the registration and test this instead.

Otherwise, it looks good to me! Really good idea and implementation, thanks :+1: